apps/spruce/src/components/styles/StyledTabs.tsx

```diff
@@ -1,10 +1,10 @@
+import { ComponentType } from "react";
 import styled from "@emotion/styled";
 import { Tabs, TabsProps } from "@leafygreen-ui/tabs";
 import { size } from "@evg-ui/lib/constants/tokens";
 
-// @ts-expect-error
 export const StyledTabs = styled(Tabs)<TabsProps>`
-  > div > [role="tablist"] {
-    margin-bottom: ${size.s};
+  [role="tabpanel"] {
+    margin-top: ${size.s};
   }
-`;
+` as ComponentType<TabsProps>;
```

Comment:
is the `as ComponentType<TabsProps>` necessary?

I think we may also want to respond on [LG-4696](https://jira.mongodb.org/browse/LG-4696). I think the typing of `setSelected` seems to be funky, not sure why they prescribe using `Dispatch<SetStateAction<number>>` and not a more flexible implementation

Reply:
Yes it is to get it to work well with Emotion. I left a comment on the slack [thread](https://mongodb.slack.com/archives/CFFA5BS79/p1734029297519359?thread_ts=1732738535.787899&cid=CFFA5BS79) irt the weird `Dispatch<SetState...` typing but I also followed up on the ticket